### PR TITLE
Fixes for Russian translation [#2]

### PR DESCRIPTION
**Pull request by Nekonyx**
- [x] accounts
- [x] admin
- [x] api
- [x] artist
- [ ] authorization
- [x] bbcode
- [x] beatmap_discussions
- [x] beatmappacks
- [ ] beatmaps
- [ ] beatmapset_events
- [ ] beatmapsets
- [ ] changelog
- [ ] common
- [ ] community
- [ ] contest
- [ ] errors
- [ ] events
- [ ] forum
- [ ] friends
- [ ] fulfillments
- [ ] help
- [ ] home
- [ ] layout
- [ ] livestreams
- [ ] model_validation
  - [ ] fulfillments
  - [ ] payments
- [ ] model_validation
- [ ] multiplayer
- [ ] news
- [ ] password_reset
- [ ] rankings
- [ ] status_page
- [ ] store
- [x] supporter_tag
- [x] user_verification
- [x] users
- [x] validation
- [x] wiki

### CODE REVIEW

File: resources/lang/ru/users.php [hunk 1]
```diff
@@ -22,19 +22,19 @@
     'deleted' => '[удалённый пользователь]',
 
     'login' => [
-        '_' => 'Войти',
-        'locked_ip' => 'Твой IP адрес заблокирован. Попробуйте ещё раз через несколько минут.',
-        'username' => 'Никнейм',
+        '_' => 'Вход',
+        'locked_ip' => 'Ваш IP адрес заблокирован. Подождите несколько минут.',
```
Comment (aplefull) [inline resources/lang/ru/users.php]: Может всё-таки лучше "твой"? Звучит более дружелюбно. (Но тогда стоило бы везде писать "твой", "тебя" итд.)

Comment (Nekonyx) [reply resources/lang/ru/users.php]: Ничего не имею против, но тогда придётся в osu-wiki заменять все "Вы" на "ты", а это признаться, очень муторно.

**Comment by Nekonyx** [comment]
wtf :thinking:

**Comment by kj415j45** [comment]
@Nekonyx ,you commit by wrong `git user.name` :p 

that's the problem

**Comment by peppy** [comment]
Rather than leaving this as a long-running open PR, how about splitting it into smaller PRs and only submitting them when you are actually done with that portion?

**Comment by Nekonyx** [comment]
I will reopen after it's ready.

**Pull request closed without merging**